src/components/Button/Button.tsx

```diff
@@ -5,6 +5,7 @@ import classNames from 'classnames';
 import { Icon } from '../Icon/Icon';
 
 export interface ButtonProps {
+	ariaLabel: string;
```

Comment:
Optional? Is enkel nodig voor buttons die geen label hebben.

Reply:
Als we dit opional maken, gaan we die labels dan niet vergeten toe te voegen?

Reply:
Het is geen kwestie van vergeten of niet, ze zijn gewoon overbodig bij 90% van de buttons.